### DISCUSSION TITLE
Globals example with securityOverwrites

### DISCUSSION THREAD

**Discussion by savage-alex** [apideck-libraries/portman#227] [Ideas]
Hi, Please could we have an example of securityOverwrites as a global override?
I have tried this using the globals example page but the security remains as defined in the openAPI
```
"globals": {
        "collectionPreRequestScripts": [
            "pm.collectionVariables.set('status', pm.iterationData.get('status') || 'open')"
          ],
          "securityOverwrites": {
            "bearer": {
              "token": "{{bearerToken}}"
            }
          }
      },
```
https://github.com/apideck-libraries/portman/blob/0f1e50c806649264a9340f94fc8defd596718b04/examples/portman-globals/readme.md

**Comment by thim81** [comment]
FYI: We added some more info on the securityOverwrites to the example:  https://github.com/apideck-libraries/portman/blob/main/examples/portman-globals/readme.md#securityoverwrites

**Comment by savage-alex** [comment]
thank you!
